### DISCUSSION TITLE
Make the font color of the popup more visible

### DISCUSSION THREAD

**Discussion by mtaishiro** [manugarg/pagenotes-issues#67] [Ideas]
Currently, the font color of the popup is very hard to see. Hope this could be changed
<img width="417" alt="スクリーンショット 2024-12-04 18 27 51" src="https://github.com/user-attachments/assets/cc70b88b-cb5a-495c-9f91-5735cea0993c">

**Comment by manugarg** [comment]
@mtaishiro That's certainly weird. I see proper font color.
<img width="408" alt="image" src="https://github.com/user-attachments/assets/b0382c85-3955-4006-94b9-a4392ceae1cc">

Maybe it's a dark mode issue. Are you using dark mode?

**Reply by mtaishiro** [reply]
Thanks for the quick reply. Yes I am using dark mode.

**Comment by manugarg** [comment]
I've created an issue from it: #69.
